cypress/support/bot.ts

```diff
@@ -54,16 +82,35 @@ Cypress.Commands.add("getBot", (synapse: SynapseInstance, displayName?: string):
 
             cli.on(win.matrixcs.RoomMemberEvent.Membership, (event, member) => {
                 if (member.membership === "invite" && member.userId === cli.getUserId()) {
-                    cli.joinRoom(member.roomId);
+                    if (opts.autoAcceptInvites) {
+                        cli.joinRoom(member.roomId);
+                    }
                 }
             });
```

Comment:
Why not wrap this whole thing in the `if` statement, so there isn't a no-op event handler when that option is set to `false`?

Reply:
Good point :+1: